modules/sync/admin-sync.php

```diff
@@ -230,4 +245,42 @@ public function __call( $func, $args ) {
 			E_USER_ERROR
 		);
 	}
+
+	/**
+	 * Returns some data (`from_post_id` and `new_lang`) from the current request.
+	 *
+	 * @since 3.7
+	 *
+	 * @param array $post_data A post array.
+	 * @return array
+	 *
+	 * @phpstan-return array{}|array{from_post_id: int<0,max>, new_lang: string}
+	 */
+	public static function get_data_from_request( array $post_data ): array {
```

Comment:
We use only the post type, so passing only the post type should be sufficient.

Reply:
Done in [8c89b5b](https://github.com/polylang/polylang/pull/1532/commits/8c89b5bf8ab7967b2a0e6841f5e98f600eee55a6).